apps/home/templates/apps_home/blocks/teaser_block_image.html

```diff
@@ -15,7 +16,9 @@ <h2 class="cta-ds-block__title">
         </h2>
         {{ value.link.specific.teaser_intro }}
         <div class="pt-5">
-            <a href="{{ value.link.url }}" class="btn btn--huge btn-outline-dark">{{ value.link_text }}</a>
+            <a href="{{ value.link.url }}" class="btn btn--huge btn-outline-dark">
+                {{ value.link_text }}
+            </a>
         </div>
     </div>
     <div class="clearfix"></div>
```

Comment:
this block will be removed

Reply:
Ah, so I should just remove it (together with the paragraph block)? Thanks!

Reply:
maybe? I was going to do a bit of a clean up now we have Marcelo's guide on zeplin, I just wanted to note it down as it didn't have the links done

Reply:
but paragraph block should definitely go!

Reply:
@fuzzylogic2000 ok looking at the zeplin thing there will be some kind of image teaser probably so leave it in, and we can clean up once designs are confirmed, what you think